### ISSUE TITLE
Critical CSS doesn't "extract" from the source file, leaving CSS to be loaded twice

### ISSUE THREAD

**Issue by hdwebpros** [bezoerb/grunt-critical#50]
Unless I'm reading into things wrong, when extracting the Critical CSS it should delete those rules from the file it grabbed them from. 

Is that the intention or does that have to be solved a different way?

**Comment by bezoerb** [comment]
Not exactly. The original css file remains untouched as the extraction may remove rules vital to subpages. Instead a copy of the original file with the extracted rules removed is created. The original file is swapped out by `inline-critical` during the inlining process.

**Comment by hdwebpros** [comment]
I'm sorry, can you please explain:
"Instead a copy of the original file with the extracted rules removed is created. The original file is swapped out by inline-critical during the inlining process."

If we are loading the critical CSS on every page and comfortable extracting that CSS from the main CSS files, what can we do to achieve that?
